### PR TITLE
Add xstack

### PR DESCRIPTION
**Pull request by privatezero**
Adds some info about xstack filter for arranging inputs

### CODE REVIEW

**Review by ablwr — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -1692,6 +1692,29 @@ <h2>Example of filter output</h2>
...
     <!-- ends Side by Side Videos/Temporal Difference Filter -->
 
+    <!-- xstack -->
+    <label class="recipe" for="xstack">Use xstack to arrange output layout of multiple video sources</label>
+    <input type="checkbox" id="xstack">
+    <div class="hiding">
+      <h3>This filter allows for the vertical and horizontal stacking of multiple video sources into one output.</h3>
+      <p>This filter is useful for the creation of output windows such as the one utilized in <a href="https://github.com/amiaopensource/vrecord">vrecord.</a></p>
+      <p><code>ffplay -f lavfi -i <em>testsrc</em> -vf "split=3[a][b][c],[a][b][c]xstack=inputs=3:layout=0_0|0_h0|0_h0+h1[out]"</code></p>
+      <p>The following example uses the 'testsrc' virtual input combined with the <a href="https://ffmpeg.org/ffmpeg-filters.html#split_002c-asplit">split filter</a> to generate the multiple inputs.
```
Comment (retokromer) [inline index.html]: the closing tag `</p>` is missing

File: index.html [hunk 2]
```diff
@@ -1692,6 +1692,29 @@ <h2>Example of filter output</h2>
...
+      <dl>
+        <dt>ffplay</dt><dd>starts the command</dd>
+        <dt>-f lavfi -i testsrc</dt><dd>tells ffplay to use the <a href="https://ffmpeg.org/ffmpeg-devices.html#lavfi" target="_blank">Libavfilter's virtual device input 'testsrc'</a></dd>
+        <dt>-vf</dt><dd>tells ffmpeg that you will be applying a filter chain to the input</dd>
+        <dt>split=3[a][b][c],</dt><dd>splits the input into three separate signals within the filter graph, named a, b and c respectively. (These are variables and any names could be used as long as they are kept consistent in following steps). The <code>,</code> separates this from the next part of the filter chain.
```
Comment (retokromer) [inline index.html]: the closing tag `</dd>` is missing

File: index.html [hunk 3]
```diff
@@ -1692,6 +1692,29 @@ <h2>Example of filter output</h2>
     </div>
     <!-- ends Side by Side Videos/Temporal Difference Filter -->
 
+    <!-- xstack -->
+    <label class="recipe" for="xstack">Use xstack to arrange output layout of multiple video sources</label>
+    <input type="checkbox" id="xstack">
+    <div class="hiding">
+      <h3>This filter allows for the vertical and horizontal stacking of multiple video sources into one output.</h3>
+      <p>This filter is useful for the creation of output windows such as the one utilized in <a href="https://github.com/amiaopensource/vrecord">vrecord.</a></p>
```
Comment (retokromer) [inline index.html]: for consistency add `target="_blank"` here and two lines below

File: index.html [hunk 4]
```diff
@@ -1692,6 +1692,29 @@ <h2>Example of filter output</h2>
     </div>
     <!-- ends Side by Side Videos/Temporal Difference Filter -->
 
+    <!-- xstack -->
+    <label class="recipe" for="xstack">Use xstack to arrange output layout of multiple video sources</label>
+    <input type="checkbox" id="xstack">
+    <div class="hiding">
+      <h3>This filter allows for the vertical and horizontal stacking of multiple video sources into one output.</h3>
```
Comment (retokromer) [inline index.html]: I am not English speaker, but would have said: `This filter allows vertical and horizontal stacking`.

Comment (privatezero) [reply index.html]: I think that 'allows for' actually conveys a particular nuance over 'allows' - BUT good point that it is a little wordy. `Enables` is more concise!

**Comment by privatezero** [comment]
thanks @retokromer!

**Review by kieranjol — APPROVED** [review]
I think this is wonderful. Your complex filter explanations are so understandable.

File: index.html (hunk 4 above)
Comment (retokromer) [reply index.html]: Thank you for the explanation.

**Review by retokromer — APPROVED** [review]

**Comment by ablwr** [comment]
Thanks @retokromer ! I trust Andrew too much.

**Comment by privatezero** [comment]
@ablwr that is def a mistake until I get some new glasses

**Comment by todrobbins** [comment]
Thank you @privatezero!

**Pull request merged**